### PR TITLE
Medical and Science Resprite

### PR DESCRIPTION
**Pull request by NonQueueingMatt**
Sprites by AmouryBlaine.

To-Do:

- [x] Testing

- [x] Add Med Beret to Loadout

- [x] Get held science obj sprite from amory

### CODE REVIEW

**Comment by alsoandanswer** [comment]
Why are the rolled down variants removed?

**Comment by alsoandanswer** [comment]
Also, tip for fixing merge conflict :

Copy paste in all new uniform sprites recently added from PR
(go to github code section > uniforms.dmi > history)

and then Resolve using 'Mine'

**Comment by NonQueueingMatt** [comment]
> 
> 
> Why are the rolled down variants removed?

They don't fit the new sprites.

**Comment by NonQueueingMatt** [comment]
!review

**Comment by NonQueueingMatt** [comment]
I swear to fucking Christ if someone uploads something that creates conflicts I'm gonna

**Comment by alsoandanswer** [comment]
Did you disable the roll down verb for those you removed the roll down sprite?

**Comment by NonQueueingMatt** [comment]
@alsoandanswer It gets disabled automatically if the code doesn't find a _d_s sprite

File: code/game/objects/structures/signs.dm [hunk 1]
```diff
@@ -133,7 +133,7 @@
 /obj/structure/sign/greencross
```
Comment (Arrow768) [inline code/game/objects/structures/signs.dm]: Change the object path aswell (and ensure to change it in all map files)

Comment (NonQueueingMatt) [reply code/game/objects/structures/signs.dm]: done

**Comment by NonQueueingMatt** [comment]
!review

**Comment by alsoandanswer** [comment]
now that i think about it blue medical is worrying

we already have bluesec, and now blue medical....

**Comment by NonQueueingMatt** [comment]
The shades of blue are really different

**Pull request merged**